odfuzz/config.py

```diff
@@ -28,6 +30,7 @@ def __init__(self):
         self._urls_per_property = int(env_url_per_property)
         env_ignore_restriction = os.getenv(ENV_IGNORE_METADATA_RESTRICTIONS,DEFAULT_IGNORE_METADATA_RESTRICTIONS)
         self._ignore_restriction = env_ignore_restriction
+        self._http_method_enabled = os.getenv(ENV_HTTP_METHOD_ENABLED, DEFAULT_HTTP_METHOD_ENABLED)
```

Comment:
remove

Reply:
cant remove the declaration. I'll set it as "GET" for now.